ldap/servers/plugins/replication/repl5_replica_config.c

```diff
@@ -1274,51 +1275,32 @@ replica_execute_ldif2cl_task(Replica *replica, char *returntext)
         goto bail;
     }
 
-    PR_snprintf(fName, MAXPATHLEN, "%s/%s.ldif", clDir, replica_get_name(replica));
+    PR_snprintf(fName, MAXPATHLEN, "%s/%s_cl.ldif", clDir, replica_get_name(replica));
 
-    rc = cl5Close();
-    if (rc != CL5_SUCCESS) {
-        PR_snprintf(returntext, SLAPI_DSE_RETURNTEXT_SIZE,
-                    "Failed to close changelog to import changelog data; "
-                    "changelog error - %d",
-                    rc);
-        slapi_log_err(SLAPI_LOG_ERR, repl_plugin_name,
-                      "replica_execute_ldif2cl_task - %s\n", returntext);
-        rc = LDAP_OPERATIONS_ERROR;
-        goto bail;
-    }
     slapi_log_err(SLAPI_LOG_INFO, repl_plugin_name,
-                  "replica_execute_ldif2cl_task -  Beginning changelog import of replica \"%s\"\n",
+                  "replica_execute_ldif2cl_task -  Beginning changelog import of replica \"%s\".  "
+                  "All replication updates will be rejected until the import is complete.\n",
                   replica_get_name(replica));
-    imprc = cl5ImportLDIF(clDir, fName, replica);
-    if (CL5_SUCCESS == imprc) {
+
+    rc = cl5ImportLDIF(clDir, fName, replica);
+    if (CL5_SUCCESS == rc) {
         slapi_log_err(SLAPI_LOG_INFO, repl_plugin_name,
                       "replica_execute_ldif2cl_task - Finished changelog import of replica \"%s\"\n",
                       replica_get_name(replica));
```

Comment:
rc = LDAP_SUCCESS so that replica_execute_ldif2cl_task returns LDAP rc

Reply:
This is fine.  CL5_SUCCESS does directly map to LDAP_SUCCESS.  If its not CL5_SUCCESS then we reset the rc to LDAP_OPERATIONS_ERROR